projects/bleach/project.yaml

```diff
@@ -0,0 +1,12 @@
+homepage: "https://github.com/mozilla/bleach"
+main_repo: "https://github.com/mozilla/bleach"
+language: python
+primary_contact: "security-tps@google.com"
```

Comment:
We can merge with this and you can subsitute contact once you find it. you can look mozilla's project.yaml for contacts.